### DISCUSSION TITLE
https://theme-hope.vuejs.press/guide/intro/intro.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#965] [Announcements]
# https://theme-hope.vuejs.press/guide/intro/intro.html

Original Intention VuePress extends Markdown syntax to a certain extent, but it still lacks some commonly used functions, such as text alignment, mark, flowchart, formula, prese...

https://theme-hope.vuejs.press/guide/intro/intro.html

**Comment by rupomthegreat** [comment]
Nice!
